spec/features/dams_object_spec.rb

```diff
@@ -685,4 +685,26 @@
     expect(page).to have_selector('#data-download-file')
     expect(page).to have_selector('#data-download-file-phone')
   end
+end
+
+#---
+
+describe 'User wants to see object view' do
+  before do
+    @unit = DamsUnit.create pid: 'xx48484848', name: "Test Unit", description: "Test Description",
+                            code: "tu", uri: "http://example.com/", group: 'dams-curator'
+    @ctsObject = DamsObject.create(pid: "xx21171293")
+    @ctsObject.damsMetadata.content = File.new('spec/fixtures/damsComplexObject1.rdf.xml').read
+    @ctsObject.save!
+    solr_index (@ctsObject.pid)
+  end
+  after do
+    @ctsObject.delete
+    @unit.delete
+  end
+  it 'with access control information on page (curator)' do
+    sign_in_developer
+    visit dams_object_path(@ctsObject.pid)
+    expect(page).to have_content('AccessCurator Only')
```

Comment:
No. It displays as "AccessCurator Only" in rspec because I am using [span] elements to style them in regular browsers.